subprojects/dependency-management/src/main/java/org/gradle/internal/resource/transfer/DefaultCacheAwareExternalResourceAccessor.java

```diff
@@ -76,8 +78,8 @@ public LocallyAvailableExternalResource getResource(final URI location, final Re
             return copyToCache(location, fileStore, delegate.withProgressLogging().getResource(location, false));
         }
 
-        // We might be able to use a cached/locally available version
-        if (cached != null && !externalResourceCachePolicy.mustRefreshExternalResource(getAgeMillis(timeProvider, cached))) {
+        // We might be able to use a cached/locally available version, when offline return the cached value
```

Comment:
Yes, please remove it from this change set. If it's worth doing, let's add as a separate PR complete with explicit test coverage.